diff --git a/node_modules/react-native-calendars/src/.DS_Store b/node_modules/react-native-calendars/src/.DS_Store
new file mode 100644
index 0000000..1b2d76c
Binary files /dev/null and b/node_modules/react-native-calendars/src/.DS_Store differ
diff --git a/node_modules/react-native-calendars/src/calendar/style.js b/node_modules/react-native-calendars/src/calendar/style.js
index c9af755..4d505bb 100644
--- a/node_modules/react-native-calendars/src/calendar/style.js
+++ b/node_modules/react-native-calendars/src/calendar/style.js
@@ -19,7 +19,8 @@ export default function getStyle(theme = {}) {
             backgroundColor: appStyle.calendarBackground
         },
         week: {
-            marginVertical: appStyle.weekVerticalMargin,
+            // marginVertical: appStyle.weekVerticalMargin,
+            marginVertical: 0,
             flexDirection: 'row',
             justifyContent: 'space-around'
         },
